repo-policy/main.tf

```diff
@@ -49,11 +49,11 @@ module "tyk-pump" {
     { branch    = "master",
       reviewers = "1",
       convos    = "false",
-    required_tests = ["1.19-bullseye", "Go 1.19 tests", "api-tests (sha256, mongo44)", "api-tests (sha256, postgres15)", "api-tests (murmur64, mongo44)", "api-tests (murmur64, postgres15)"] },
+    required_tests = ["1.21-bullseye", "Go 1.21 tests", "api-tests (sha256, mongo44)", "api-tests (sha256, postgres15)", "api-tests (murmur64, mongo44)", "api-tests (murmur64, postgres15)"] },
     { branch        = "release-1.8",
       reviewers     = "0",
       convos        = "false",
       source_branch = "master",
-    required_tests = ["1.19-bullseye", "Go 1.19 tests"] },
+    required_tests = ["1.21-bullseye", "Go 1.21 tests"] },
```

Comment:
@mativm02  are you planing to merge this to release-1.8 ? if we apply terraform PRs will be blocked until you modify the workflows with the proper go versions

Reply:
@mativm02 let's remove this since Go 1.21 it's not required and not gonna be propagated to 1.8.x